index.js

```diff
@@ -1,4 +1,94 @@
-//  in this file create an express application - use the middle-ware built into express
-//  to serve up static files from the public directory (index.html and client.js - you 
-//  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks/:name'
\ No newline at end of file
+'use strict';
+//  in this file create an express application
+//  use the middle-ware built into express
+//  to serve up static files from public directory (index.html & client.js)
+//  you can also serve up css files from the public directory  if you want)
+//  you need to support '/trucks' endpoint
+// and dynamic route for '/trucks/:name'
+
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+var express = require('express');
+var trucks = require('./trucks');
+
+/* eslint-enable no-unused-vars */
+/* eslint-enable no-undef */
+
+var app = express();
+var serveStatic = express.static('public');
+app.use(serveStatic);
+
+/* eslint-disable no-console */
+
+app.use(function (request, response, next) {
+  console.log('Request URL: ', request.originalURL);
+  next();
+});
+
+app.use(function (request, response, next) {
+  console.log('Time: ', Date.now());
+  next();
+});
+
+//   `/trucks` This route returns the list of all trucks in the module.
+// http://127.0.0.1:3000/trucks
+app.get('/trucks', function (request, response, next) {
+  var truckList = trucks.getTrucks();
+  response.send(truckList);
+  next();
```

Comment:
Routes handlers don't need next as a callback parameter, nor should they call it after they've sent a response.